README.md

```diff
@@ -105,58 +90,52 @@ startup phase, they cannot be completely ruled out.
 
 ## Troubleshooting
 
-- may require extra packages on Linux
-
-  ```bash
-  apt update
-  apt install libdbus-1-3
```

Comment:
I needed to install gnome-keyring, but you seem to be right that we don't really need dbus, which is neat because it makes things a lot simpler!